src/util/get-electron-version.js

```diff
@@ -0,0 +1,28 @@
+import debug from 'debug';
+import path from 'path';
+import readPackageJSON from './read-package-json';
+
+const d = debug('electron-forge:util');
+
+export default async (projectDir) => {
+  let result = null;
+
+  const modulesToExamine = ['electron-prebuilt-compile', 'electron-compile', 'electron'];
+  for (let i = 0; i < modulesToExamine.length; i += 1) {
```

Comment:
`for...of`?

Also @MarshallOfSound this would be interesting if we gather this info when `flora-colossus` walks the module tree.